cas-client-core/pom.xml

```diff
@@ -10,8 +10,37 @@
     <packaging>jar</packaging>
     <name>Jasig CAS Client for Java - Core</name>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
     <dependencies>
         <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Moving to saml module produces compilation errors:

    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.0:compile (default-compile) on project cas-client-core: Compilation failure: Compilation failure:
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/session/SingleSignOutHandler.java:[31,31] package org.apache.commons.lang does not exist
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/configuration/BaseConfigurationStrategy.java:[21,31] package org.apache.commons.lang does not exist
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/configuration/BaseConfigurationStrategy.java:[22,36] package org.apache.commons.lang.math does not exist
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/session/SingleSignOutHandler.java:[328,13] cannot find symbol
    [ERROR] symbol:   variable StringUtils
    [ERROR] location: class org.jasig.cas.client.session.SingleSignOutHandler
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/configuration/BaseConfigurationStrategy.java:[41,24] cannot find symbol
    [ERROR] symbol: variable BooleanUtils
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/configuration/BaseConfigurationStrategy.java:[49,24] cannot find symbol
    [ERROR] symbol: variable NumberUtils
    [ERROR] /Users/marvin/Code/java-cas-client/cas-client-core/src/main/java/org/jasig/cas/client/configuration/BaseConfigurationStrategy.java:[57,24] cannot find symbol
    [ERROR] symbol: variable NumberUtils

My guess is that we got commons-lang transitively in core via opensaml, so we have to explicitly declare it now that opensaml is gone from core.